### ISSUE TITLE
Evaluation Setting

### ISSUE THREAD

**Issue by DorLitvak** [ThunderVVV/HaWoR#3]
Hi! :)  

Would you be able to share the evaluation code and settings? 
In addition, the paper mentions using 110 videos as the training set and 27 videos as the validation set from HOT3D. 
Would you be able to share the exact training and evaluation subset? 

Thank you so much!

**Comment by xiaoyudanaa** [comment]
> Hi! :)
> 
> Would you be able to share the evaluation code and settings? In addition, the paper mentions using 110 videos as the training set and 27 videos as the validation set from HOT3D. Would you be able to share the exact training and evaluation subset?
> 
> Thank you so much!

I need it too, can I talk to you first, my email is [xiaoyudan@tju.edu.cn](mailto:xiaoyudan@tju.edu.cn)
